code/components/citizen-resources-core/include/StateBagComponent.h

```diff
@@ -72,7 +72,7 @@ class CRC_EXPORT StateBag : public std::enable_shared_from_this<StateBag>
 	//
 	// Sets data for a key.
 	//
-	virtual void SetKey(int source, std::string_view key, std::string_view data, bool replicated = true) = 0;
+	virtual void SetKey(int slotId, int source, std::string_view key, std::string_view data, bool replicated = true) = 0;
```

Comment:
Just a note for me later, if people need this they can just get the entity owner, this adds a lot of mess